core/templates/dev/head/pages/library/LibraryFooter.js

```diff
@@ -16,8 +16,20 @@
  * @fileoverview Data and controllers for the Oppia library footer.
  */
 
+// The constants defined below in LIBRARY_PAGE_MODES should be same as the
```

Comment:
Actually, drop this and modify the other comments to exclude reference to LibraryFooter.js. I forgot that all frontend files will get concatenated, so we don't really need to declare this twice with one overwriting the other. Sorry!